google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java

```diff
@@ -484,7 +484,7 @@ private <RowT> UnaryCallable<Query, List<RowT>> createBulkReadRowsCallable(
         new TracedBatcherUnaryCallable<>(readRowsUserCallable.all());
 
     UnaryCallable<Query, List<RowT>> withBigtableTracer =
-        new BigtableTracerUnaryCallable<>(tracedBatcher);
+        new BigtableTracerUnaryCallable<>(tracedBatcher, true);
```

Comment:
using a boolean here is hard to read...consider using 2 different factory methods:
BigtableTracerUnaryCallable.createForBatchedCallable

Reply:
Better yet, make a separate BigtableTracerBatchedCallable that inherits from the Unary one